build_tools/check.sh

```diff
@@ -35,6 +35,10 @@ if $lint; then
 fi
 
 repo_root="$(dirname "$0")/.."
+# The directory containing the binaries produced by cargo/rustc.
+# Currently, all builds are debug builds.
+# Note that `$target_triple` might be empty, resulting in `//debug` which is interpreted as
+# `/debug`.
 build_dir="${CARGO_TARGET_DIR:-$repo_root/target}/${target_triple}/debug"
```

Comment:
Yeah this is weird, especially because in 90% of all cases we are only aware of target/,
as opposed to target/x86_64-linux-gnu/.
The line is also a bit long, we could be explicit about how this works:

	target_dir=${CARGO_TARGET_DIR:-$workspace_root/target}
	if [ -n "$target_triple" ]; then
	    target_dir=$target_dir/$target_triple
	fi
	build_dir=$target_dir/debug

The comment is fine too, your call.